intake/catalog/local.py

```diff
@@ -578,3 +623,7 @@ def plugins(self):
     @property
     def entries(self):
         return self._entries
+
+    @property
+    def cats(self):
```

Comment:
Can we just call them `catalogs`?

Reply:
Sure! I was thinking that the PR here gives a possible outline, but not actually the right way to do things, so I wanted a conversation around this.

Reply:
Done